packages/venia-concept/src/reducers/purchaseDetails.js

```diff
@@ -0,0 +1,34 @@
+import { handleActions } from 'redux-actions';
+
+import actions from 'src/actions/purchaseDetails';
+
+export const name = 'purchaseDetails';
+
+const initialState = {
+    item: null,
+    otherItems: null,
+    isFetching: false
+};
+
+const reducerMap = {
+    [actions.request]: () => {
+        return {
```

Comment:
please, destructure all state, just for the case when we add some other properties to our model.
```
return {
  ...state,
  isFetching: true
}
```